src/app/rundown/components/timeline-markers/timeline-markers.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import { TestBed } from '@angular/core/testing'
 
 import { TimelineMarkersComponent } from './timeline-markers.component'
 import { instance, mock } from '@typestrong/ts-mockito'
-import { MillisecondsAsTimePipe } from '../../../shared/pipes/millisecondsAsTime.pipe'
+import { TimestampPipePipe } from '../../../shared/pipes/timestamp.pipe'
```

Comment:
PipePipe? :D

Reply:
It is fixed in https://github.com/tv2/ng-sofie/pull/29.